homeassistant/components/idasen_desk/config_flow.py

```diff
@@ -65,14 +65,14 @@ async def async_step_user(
             desk = Desk(None, monitor_height=False)
             try:
                 await desk.connect(discovery_info.device, auto_reconnect=False)
-            except AuthFailedError as err:
-                _LOGGER.exception("AuthFailedError", exc_info=err)
+            except AuthFailedError:
+                _LOGGER.exception("Authentication failed")
                 errors["base"] = "auth_failed"
-            except TimeoutError as err:
-                _LOGGER.exception("TimeoutError", exc_info=err)
+            except TimeoutError:
+                _LOGGER.exception("Connect timeout")
                 errors["base"] = "cannot_connect"
-            except BleakError as err:
-                _LOGGER.exception("BleakError", exc_info=err)
+            except BleakError:
+                _LOGGER.exception("Unexpected Bluetooth error")
```

Comment:
On the fence here if we should log the trace or not since it may have more useful info

Reply:
true